internal/js/modules/k6/grpc/client.go

```diff
@@ -264,6 +265,14 @@ func (c *Client) Connect(addr string, params sobek.Value) (bool, error) {
 		opts = append(opts, grpc.WithAuthority(p.Authority))
 	}
 
+	if p.Compression.Name != "" {
```

Comment:
```suggestion
	if p.Compression != nil && p.Compression.Name != "" {
```
This is causing panic in the tests.